frontend/src/actions/WindowActions.js

```diff
@@ -1201,17 +1210,13 @@ export function handleProcessResponse(response, type, id, parentId) {
               keepProcessModal = true;
 
               await dispatch(
-                openModal(
-                  '',
-                  action.windowId,
-                  'window',
-                  null,
-                  null,
-                  action.advanced ? action.advanced : false,
-                  '',
-                  '',
-                  action.documentId
-                )
+                openModal({
+                  title: '',
```

Comment:
Should this be passed as param in here if it already has the same default value ?

Reply:
Good point